lib/future.js

```diff
@@ -0,0 +1,51 @@
+/* global Promise:true */
```

Comment:
We shouldn't need this "future" stuff anymore, since this whole package is using ES6/7. In the `authorize.js` file you ought to be able to just import `xcode` and use it as you are.